src/main/java/io/github/externschool/planner/controller/UserController.java

```diff
@@ -0,0 +1,34 @@
+package io.github.externschool.planner.controller;
+
+import io.github.externschool.planner.dto.UserDTO;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+public class UserController {
+
+    //TODO remove this method when any page that requires authorisation is available, please
+    //TODO "success" form is now accessible with mapping only in test purpose
+    @RequestMapping("/success")
+    public String homeUser(Model model){
+        model.addAttribute("user", new UserDTO());
+        return "success";
+    }
+
+    //TODO Add service to prevent sign up for logged in users
+    @RequestMapping(value = "/signup", method = RequestMethod.GET)
+    public String newUser(Model model){
+        model.addAttribute("user", new UserDTO());
+        return "signup";
+    }
+
+    //TODO Browser-side SignUp form validation used
+    //TODO Implement DTO validation to avoid fake data submitted in POST requests
+    @RequestMapping(value = "/signup", method = RequestMethod.POST)
+    public String saveOrUpdate(UserDTO userDTO, Model model) {
+        model.addAttribute("user", userDTO);
```

Comment:
Maybe it's better split save/updete methods and call service.createNewUser?

Reply:
this PR realizes the task #21:
* Add UserDto with fields that exist on registration form. UserDto should have package dto that shoud be placed on main package our apllication.
* Add UserController with method that handle GET request for url /registration and returns template with user registration form. Controller should return model with attribute that be new instance of UserDto class
* On login form add link to registration page